c/meterpreter/workspace/meterpreter.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
looks like this inadvertently downgraded the meterpreter sln file, no worries, I can fix it up.